ansible/roles/collectory/templates/config/generic-collectory-config.properties

```diff
@@ -0,0 +1,73 @@
+# CAS Config
```

Comment:
I don't think we need to have a specific file for the generic collectory. The base template should be sufficient for any collectory implementation: anything that needs to change between implementations (e.g. URLs) would be configured in the inventory file.